frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -140,6 +143,28 @@ func NewCVDToolInstanceManager(opts *CVDToolInstanceManagerOpts) *CVDToolInstanc
 	}
 }
 
+func newCVDExecContext(execContext ExecContext, user string) CVDExecContext {
```

Comment:
let's move `user` as the first parameter and and emphasizes the big difference between passing a value and not in the comments.

Reply:
The execContext parameter is arguably more important since it's being wrapped by the result, while the user is just a customization parameter.

Reply:
true.